ros/lib/constants.py

```diff
@@ -32,3 +27,57 @@ class CloudProvider(Enum):
 class OperatingSystem(Enum):
     """Supported Operating Systems"""
     RHEL = 'RHEL'
+
+
+class SystemStatesWithKeys(Enum):
+    SYSTEM_STATES = {
+        "INSTANCE_OVERSIZED": "Oversized",
+        "INSTANCE_UNDERSIZED": "Undersized",
+        "INSTANCE_IDLE": "Idling",
+        "INSTANCE_OPTIMIZED_UNDER_PRESSURE": "Under pressure",
+        "STORAGE_RIGHTSIZING": "Storage rightsizing",
+        "OPTIMIZED": "Optimized",
+        "NO_PCP_DATA": "Waiting for data"
+    }
+    OPTIMIZED_SYSTEM = "OPTIMIZED"
```

Comment:
Do we require dict in Enum? as example,
~~~
>>> class TestEnum(Enum):
...     CAT="TOM"
...     DOG="GOOFY"
... 
>>> TestEnum.CAT
<TestEnum.CAT: 'TOM'>
>>> TestEnum.DOG
<TestEnum.DOG: 'GOOFY'>
>>> TestEnum.__members__.values()
dict_values([<TestEnum.CAT: 'TOM'>, <TestEnum.DOG: 'GOOFY'>])
>>> TestEnum._member_names_
['CAT', 'DOG']
~~~